Thursday

jwileman1 opened the pull request:
Imagery date updates (again...)- sorry- I figured out how to 'fix' other branch after i had fixed in this branch 
added details from capture rules document to a couple feature classes- minor additions that were missing
added cantilevered building subtype ... including photo ...   Question:  do you have a Feature View ?
added BIN attribute to Cooling Tower

mattyschell commented:
very thorough, looks great.

I'm no sure what you mean by this:
> Question: do you have a Feature View ?

jwileman1 commented:
when I added cant. bldg subtype ... the format from all other subtypes had a Feature View 'graphic' that showed an example of a feature overlaid on imagery.  The Capture Rules has most of these 'graphics' but not one for cant. bldg - it did have a photo that I added already but no feature view ('graphic').

________________________________
From: Matt Schell ***@***.***>
Sent: Friday, December 29, 2023 11:10 AM
To: CityOfNewYork/nyc-planimetrics ***@***.***>
Cc: John Wileman ***@***.***>; Author ***@***.***>
Subject: Re: [CityOfNewYork/nyc-planimetrics] Thursday (PR #23)


very thorough, looks great.

I'm no sure what you mean by this:

Question: do you have a Feature View ?

—
Reply to this email directly, view it on GitHub [github.com]<https://urldefense.com/v3/__https://github.com/CityOfNewYork/nyc-planimetrics/pull/23*issuecomment-1872194020__;Iw!!CKZwjTOV!xcRYwX87sWuYAF5Rv0abJnCWPdVawaTjsxv3IeXDpoDdXZo02HQROHfjQu-mZj8If737fAbRhFb2iR1iYcXu1A$>, or unsubscribe [github.com]<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/BE2LE77RPWM3A27QSFQOCM3YL3TPLAVCNFSM6AAAAABBGYEOB6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQNZSGE4TIMBSGA__;!!CKZwjTOV!xcRYwX87sWuYAF5Rv0abJnCWPdVawaTjsxv3IeXDpoDdXZo02HQROHfjQu-mZj8If737fAbRhFb2iR3a68SYTw$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

mattyschell commented:
Aha, I see what you mean.  No I don't think we have a feature view for cantilevered buildings yet.

I'll file an issue to return to this since you are right, it will look a little weird to have just one without the graphic.  But I think we can skip it for now.
